powersimdata/design/generation/curtailment.py

```diff
@@ -0,0 +1,124 @@
+import pandas as pd
+
+from powersimdata.scenario.scenario import Scenario
+
+default_pmin_dict = {
+    "coal": None,
+    "dfo": 0,
+    "geothermal": 0.95,
+    "hydro": None,
+    "ng": 0,
+    "nuclear": 0.95,
+    "other": 0,
+    "solar": 0,
+    "wind": 0,
+    "wind_offshore": 0,
+}
+profile_methods = {
+    "hydro": "get_hydro",
+    "solar": "get_solar",
+    "wind": "get_wind",
+    "wind_offshore": "get_wind",
+}
```

Comment:
Should we write a `get_onshore_wind` and `get_offshore_wind` methods in the `Analyze` class that would relieve our brain once for all?

Reply:
We need this to work for Create as well.

Reply:
Indeed. This thing is hunting me.